tools/make-config.py

```diff
@@ -1016,8 +1065,59 @@ def hydrate_psp_symbols(splat_config_path: str, splat_config, version: str):
         spinner_start("updating all symbols")
         split(splat_config_path, True)
 
-
-def hydrate_psx_duplicate_symbols(splat_config, ovl_name: str, version: str):
+def hydrate_psx_matching_symbols(splat_config, ovl_name: str, version: str, options: Options):
+    spinner_start("finding matches across overlays")
+    seen_functions = set()
+
+    # find well-known segment offsets
+    for source_name, source_bin in {
```

Comment:
Wherever these values are should be ordered consistently with what we see elsewhere: game - stage - boss - servant, with reverse stages and bosses following normal stages and bosses respectively.